scss/_tables.scss

```diff
@@ -34,6 +34,10 @@
     vertical-align: inherit;
   }
 
+  tbody + tbody {
+    border-top: (1.1 * $table-border-width) solid;
```

Comment:
Why `1.1` here?

Reply:
because you need to give more value from initial value
I've testing the best result is `1.1` when boostrap 4 give` 2 * $table-border-width`

on bootstrap 4 you have tbody + tbody like this
![b6](https://user-images.githubusercontent.com/37553901/131607835-204a9d4e-fe35-41c1-bd6c-99af055482f7.png)

but not on bootstrap 5
and the problem from this #34732 is missing feature  when use boostrap 5  if you use
```
<tbody>
....
</tbody>
<tbody>
....
</tbody>
```

Reply:
As explained in the issue, this is replaced with a more generic selector. However it's targetting last children and uses bottom border, which leads to the issue.

Using this would help reduce the gap with v4, you're right.

I'll re-open this, but 1.1 feels like a magic number and doesn't make sense. We need to try:
- the initial value, 
- the v4 value (`*2`)
- and maybe simply add a single px would work, using `add()`? The intention would be more understandable this way, IMHO.

Thanks again for your involvement, this is highly appreciated!

Reply:
yes it is like magic.
But since the initial value is 1 this will be fine without changing any other code and adding 0.1 this is just a requirement to make the second border much bigger than the first border.
if you try to go back to 2 it's worse because the gaps 1 and 2 are far away, and you have to change some code. I don't know why, but since v5 the border was reduced to 1 instead of 2. this just IMO 😄